sites/main-site/src/content/about/governance.mdx

```diff
@@ -182,6 +183,7 @@ The outreach leads will have access to community social media and YouTube accoun
 <Profile username="yuukiiwa">Yuk Kei</Profile>
 <Profile username="vdauwera">Geraldine Van der Auwera</Profile>
 <Profile username="chriswyatt1">Chris Wyatt</Profile>
+<Profile username="kenibrewer">Ken Brewer</Profile>
```

Comment:
```suggestion
```

Trying to kick me out so soon? 😜

Reply:
Kick you out? I was adding you to outreach - I noticed you weren't there already and assumed that it was in error?

Reply:
(in addition to adding you to core team above)

Reply:
Ohhh.... That was me not reading the PR carefully. Looked like I was getting added to Alumni 😅 
![image](https://github.com/user-attachments/assets/f85958ca-a3be-4caf-8273-72861190ef0e)

Reply:
![](https://frinkiac.com/video/S08E05/BFlb7KPKAaT78jc3ppl482zgfqQ=.gif)